src/views/virtualmachinesinstances/utils/virtualMachinesInstancesRowFilter.ts

```diff
@@ -0,0 +1,51 @@
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+import { OS_TEMPLATE_LABEL } from './virtualMachinesInstancesConstans';
+import { osNames, vmiStatuses } from './virtualMachinesInstancesStatuses';
+
+const makeUserTitleOSName = (osName: string): string => {
+  //osName is in format: rhel8.5
+  const title = osName.split(/(\d.*)/, 2).join(' ');
+  //Will produce Rhel 8.5
+  console.log('title: ', title.charAt(0).toUpperCase() + title.slice(1));
+  return title.charAt(0).toUpperCase() + title.slice(1);
+};
+
+const getOSName = (obj: V1VirtualMachineInstance): string | undefined => {
```

Comment:
why not just `vmi?.metadata?.annotations?.vm.kubevirt.io/os` ?

Reply:
replaced to it, good idea